volumes/vds/sql/tables/create-table-detector_inventory_cursed.sql

```diff
@@ -0,0 +1,56 @@
+DROP TABLE vds.detector_inventory_cursed;
```

Comment:
I think these `DROP` statements are dangerous unless you added it intentionally. Is it safe to keep this query in this file?